src/WInterop.Desktop/Windows/Native/Imports.UxTheme.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Jeremy W. Kuhne. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+using WInterop.Errors;
+using WInterop.Support;
+
+namespace WInterop.Windows.Native
+{
+    /// <summary>
+    ///  Direct usage of Imports isn't recommended. Use the wrappers that do the heavy lifting for you.
+    /// </summary>
+    public static partial class Imports
```

Comment:
I've started naming the `Imports` more specifically. So this is now `WindowsImports`. I had put the rest of the uxtheme ones in the main `WindowsImports` file, but I'm fine with having them grouped in a separate file. You can pull the existing ones into here or go in the main one, I don't care.